src/commands/start.command.ts

```diff
@@ -41,6 +42,10 @@ export const start = async (pmConfigFilePath: string) => {
     processes[app.name] = p;
   }
 
+  try {
+    unlinkSync("./.pm.sock");
```

Comment:
I wasn't able to check if the process is running so I opted for the error message
It would be nice though if it would be handle automatically somehow.

Reply:
:+1: we can iterate on that when we have more experience and feedback (how often this issue happens etc)

one thing though: please remove the try-catch, that is now useless. (BTW: silent catch is in most cases a bad idea)

Reply:
I removed the try-catch